src/lib/profiles/data-management/Current/GenericTraitCatalogImp.h

```diff
@@ -0,0 +1,535 @@
+/*
+ *
+ *    Copyright (c) 2019 Google, Inc.
```

Comment:
See other files; officially it is Google LLC.

Reply:
sure, I would update the description, this would be used in data management support on mobile client side